### PR TITLE
211 multiselector bdd implementation

### PR DESCRIPTION
**Pull request by Sudalius**
Implemented BDD steps for Multiselector

### CODE REVIEW

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        multiSelector.assertThat().values(containsInAnyOrder(LinqUtils.toStringArray(values)));
+    }
+
+    @When("^I select (.*) numbers in \"([^\"]*)\" multiselector$")
+    public void selectValuesByNumber(List<Integer> values, String name) {
+        MultiSelector multiSelector = getUI(name, MultiSelector.class);
+        multiSelector.check(LinqUtils.toIntArray(values));
+    }
+
+    @Then("^I check the \"([^\"]*)\" value is selected in \"([^\"]*)\" multiselector")
```
Comment (ChepelVE) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java]: add $ at the end

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import java.util.List;
+
+import static com.epam.jdi.bdd.Utils.getUI;
+import static org.hamcrest.Matchers.*;
+import static org.testng.Assert.assertEquals;
+
+public class MultiSelectorSteps {
+
+    @When("^I select \"([^\"]*)\" with value \"([^\"]*)\"$")
```
Comment (ChepelVE) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java]: add : to the end of all steps where input as gherkin table

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        multiSelector.is().selected(value);
+    }
+
+    @Then("^I check the \"([^\"]*)\" multiselector contains \"([^\"]*)\" value$")
+    public void assertMultiselectorValue(String name, String value) {
+        MultiSelector multiSelector = getUI(name, MultiSelector.class);
+        multiSelector.is().values(hasItem(value));
+    }
+
+    @Then("^I check the \"([^\"]*)\" multiselector in disabled state has \"([^\"]*)\" item$")
```
Comment (ChepelVE) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java]: change all steps by analogy with
@Then("^the \"([^\"]*)\" multiselector has \"([^\"]*)\" disabled$")

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java (hunk 2 above)
Comment (Sudalius) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiSelectorSteps.java]: In this step I don't see a gherkin table, but I have ":" in other steps

**Pull request merged**